docs/changelog.md

```diff
@@ -283,6 +283,9 @@ future and `StringSlice.__len__` now does return the Unicode codepoints length.
   # pw_gecos='System Administrator', pw_dir='/var/root', pw_shell='/bin/zsh')
   ```
 
+- `bit` module now supports `bit_reverse()`, `byte_swap()` and `pop_count()` for
```

Comment:
Please make sure you give yourself credit here :)

Reply:
Got it. Thanks for the reminder!